frontend/public/components/monitoring/_monitoring.scss

```diff
@@ -437,6 +437,10 @@ $monitoring-line-height: 18px;
   }
 }
 
+.query-browser__series-select-all-btn {
+  padding-top: 15px !important;
```

Comment:
We try hard to avoid using `!important`. Can we use a more specific CSS rule like
```
.query-browser__table-wrapper .query-browser__series-select-all-btn {
  padding-top: 15px;
}
```
Or something similar?

Reply:
👍 Updated in the latest commit : 
https://github.com/openshift/console/blob/c5e2e41052f4a74edbd4841b5369e458843965d1/frontend/public/components/monitoring/_monitoring.scss#L498